.github/workflows/main.yml

```diff
@@ -38,6 +38,14 @@ jobs:
         run: |
            mkdir -p ~/.ghidra/.ghidra_${{ matrix.ghidra }}_PUBLIC/Extensions/
            unzip Ghidra-Cpp-Class-Analyzer/dist/*.zip -d ~/.ghidra/.ghidra_${{ matrix.ghidra }}_PUBLIC/Extensions/
+
+      - uses: actions/upload-artifact@v2
+        if: github.repository == 'astrelsky/GhidraOrbis' && github.event_name != 'pull_request'
+        with:
+          name: GhidraOrbis
+          path: dist/*.zip
+          if-no-files-found: warn
```

Comment:
This should either be changed to `error` or removed.

Reply:
Lets get the build passed on PR first 😄